cmd/up/up.go

```diff
@@ -227,7 +227,7 @@ func Up() *cobra.Command {
 			}
 			up.Client, up.RestConfig, err = okteto.GetK8sClient()
 			if err != nil {
-				return fmt.Errorf("failed to load okteto context '%s': %v", up.Dev.Context, err)
+				return fmt.Errorf("failed to load okteto context: %v", err)
```

Comment:
```suggestion
				return fmt.Errorf("failed to load k8s client: %v", err)
```

the error could be the context load or getting the client, i think is better to change the string of the error too. whatever is returned by `okteto.GetK8sClient` will be added to the error.